site/ko/tutorials/text/text_classification_rnn.ipynb

```diff
@@ -0,0 +1,723 @@
+        "colab": {},
+        "colab_type": "code",
+        "id": "EplYp5pNnW1S"
+      },
+      "outputs": [],
+      "source": [
+        "encoder = info.features['text'].encoder"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "e7ACuHM5hFp3"
+      },
+      "outputs": [],
+      "source": [
+        "print ('단어 크기: {}'.format(encoder.vocab_size))"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "tAfGg8YRe6fu"
+      },
+      "source": [
+        "이 텍스트 인코더는 어떠한 문자열도 되돌릴 수 있게 코딩하여 만약 필요로 한다면 바이트 인코딩으로 돌아갈 수 있습니다."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "Bq6xDmf2SAs-"
+      },
+      "outputs": [],
+      "source": [
+        "sample_string = 'Hello TensorFlow.'\n",
+        "\n",
+        "encoded_string = encoder.encode(sample_string)\n",
+        "print ('인코딩된 문자열 {}'.format(encoded_string))\n",
+        "\n",
+        "original_string = encoder.decode(encoded_string)\n",
+        "print ('원래 문자열 \"{}\"'.format(original_string))"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "TN7QbKaM4-5H"
+      },
+      "outputs": [],
+      "source": [
+        "assert original_string == sample_string"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "MDVc6UGO5Dh6"
+      },
+      "outputs": [],
+      "source": [
+        "for index in encoded_string:\n",
+        "  print ('{} ----\u003e {}'.format(index, encoder.decode([index])))"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "GlYWqhTVlUyQ"
+      },
+      "source": [
+        "## 훈련을 위한 데이터 준비"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "z2qVJzcEluH_"
+      },
+      "source": [
+        "다음으로 이러한 인코딩 된 문자열을 일괄적으로 생성합니다. `padded_batch` 메소드를 사용하여 배치를 가장 긴 문자열 n의 길이로 시퀀스를 zero-pad합니다:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "dDsCaZCDYZgm"
+      },
+      "outputs": [],
+      "source": [
+        "BUFFER_SIZE = 10000\n",
+        "BATCH_SIZE = 64"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "VznrltNOnUc5"
+      },
+      "outputs": [],
+      "source": [
+        "train_dataset = train_dataset.shuffle(BUFFER_SIZE)\n",
```

Comment:
Notebook is throwing an error:
`AttributeError: 'ShuffleDataset' object has no attribute 'output_shapes'`

Can you double-check this notebook against the one in en/ ? Thanks

Reply:
Oh, it's because of the version issue. can you close this PR? I'll make new PR for this document(there is no response from a translator.. :))